### DISCUSSION TITLE
Local Subnets are unreachable after enabling IPSEC

### DISCUSSION THREAD

**Discussion by SanthoshNetcon** [strongswan/strongswan#1627] [Q&A]
Consider the below Setup diagram.
![image](https://user-images.githubusercontent.com/69035516/227788965-b05bf4b6-6ab3-4a94-900b-cb5ba2580cd2.png)

We are using strongswan / swanctl version of **5.8.2** and Ubuntu Version **20.04** 
Connections between the Local to Remote is established successfully using the below configuration.

**/etc/swanctl/conf.d/my_connection.conf**
```
connections{
   my-conn{
          local_addrs = 20.10.xx.xx
          remote_addrs = 30.10.xx.xx
          proposals = aes128-sha1-modp1024
          version = 2
          local {
              auth = psk
          }
          remote {
              auth = psk
          }
          children{
              my-conn{
                  local_ts = 172.168.10.0/24
                  remote_ts = 0.0.0.0/0
                  start_action = start
                  esp_proposals = aes128-sha1-modp1024
                  dpd_action = restart
               }
          }
    }
}
secrets{
      ike-my-conn{
                secret = "*********"
      }
}
```
The above configuration successfully creates the IPSEC connection. But once the IPSEC connection gets enabled, I can't reach machines in the local subnet
Example: I tried to ping from **172.168.10.10** to **172.168.10.11** the packet is not reachable. Even I tried to debug using the tcpdump on the Local Subnet of the Gateway. I can see the ping request on the 172.168.10.1 machine, of the packet from 172.168.10.10 to 172.168.10.11. But after that, I can't see the reply. I felt that the ping reply packet goes through the IPSEC tunnel.

How to rectify this issue? What is the proper way of configuration for this setup? How to force route this?

**Comment by tobiasbrunner** [comment]
Please provide more information (logs, status output, see #196).

**Comment by drcyberg** [comment]
Hi!
I had the same problem but this solved it: [bypass-lan](https://docs.strongswan.org/docs/5.9/plugins/bypass-lan.html)
